builder/source/simple.go

```diff
@@ -20,14 +20,15 @@ import (
 	"crypto/sha1"
 	"crypto/sha256"
 	"encoding/hex"
-	"fmt"
-	log "github.com/DataDrake/waterlog"
-	curl "github.com/andelf/go-curl"
-	"github.com/cheggaaa/pb/v3"
 	"io/ioutil"
 	"net/url"
 	"os"
 	"path/filepath"
+	"time"
+
+	log "github.com/DataDrake/waterlog"
+	grab "github.com/cavaliergopher/grab/v3"
```

Comment:
Renaming the package into "grab" is unnecessary, Go understands that v3 is the version and not the package name.